### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by vagkos1**

### CODE REVIEW

File: 3-tutorial.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ have installed.
 - [Creating a Deployment](#creating-a-deployment)
 - [Making Your Deployment Accessible](#making-your-deployment-accessible)
     - [Exposing Ports](#exposing-ports)
+        - [Communication in a Pod](#communication-in-a-pod)
```
Comment (REBELinBLUE) [inline 3-tutorial.md]: What about the `toc.md`? :p

File: 3-tutorial.md [hunk 2]
```diff
@@ -611,7 +618,7 @@ The cluster now looks like this.
 ![Service](./images/figure4.png)
 
 You may notice the *Endpoint*, just like *Deployments* create *ReplicaSets* which point at the specific *Pods*, 
-*Services* are backed by *Endpoints* which point to the *Pods*. You can see how this looks (bare in mind your output
+*Services* are backed by *Endpoints* which point to the *Pods*. You can see how this looks (bear in mind your output
```
Comment (REBELinBLUE) [inline 3-tutorial.md:621]: 😱

File: 3-tutorial.md (hunk 1 above)
Comment (vagkos1) [reply 3-tutorial.md]: added

File: toc.md [hunk 3]
```diff
@@ -62,6 +62,8 @@
     - [Creating a Deployment](./3-tutorial.md#creating-a-deployment)
     - [Making Your Deployment Accessible](./3-tutorial.md#making-your-deployment-accessible)
         - [Exposing Ports](./3-tutorial.md#exposing-ports)
+            - [Communication in a Pod](#communication-in-a-pod)
```
Comment (REBELinBLUE) [inline toc.md]: Going to be picky... `in a Pod` should actually be `within a Pod`

Comment (vagkos1) [reply toc.md]: I prefer your wording, one sec

Comment (vagkos1) [reply toc.md]: fixed

**Pull request merged**